__tests__/components/load/LoadResource.test.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 Stanford University see LICENSE for license
```

Comment:
2019!

Reply:
Doh! No more PR review before coffee for me. Missed that.